src/core/turn-order.js

```diff
@@ -48,6 +48,22 @@ export const TurnOrder = {
     next: (G, ctx) => (+ctx.currentPlayer + 1) % ctx.numPlayers + '',
   },
 
+    /**
+   * DEFAULT-REVERSE
+   *
+   * The default round-robin turn order starting from the last player.
+   */
+  DEFAULT_REVERSE: {
+    first: (G, ctx) => ctx.numPlayers -1 + '',
+    next: (G, ctx) => {
```

Comment:
This should also work:
(+ctx.currentPlayer + ctx.numPlayers - 1) % ctx.numPlayers + ''

Reply:
yes, this is cleaner